code/datums/wounds/robotic/robotic_blunt.dm

```diff
@@ -0,0 +1,1027 @@
+	var/head_movement_shake_chance = 100
+	/// The base chance, in percent, for moving to increase dizziness. Multiplied against many local modifiers, such as resting, being gauzed, etc.
+	var/head_movement_dizzy_chance = 100
+
+	/// The base duration, in deciseconds, for our camera shake on movement.
+	var/head_movement_base_shake_duration = 1
+	/// The base intensity, in tiles, for our camera shake on movement.
+	var/head_movement_base_shake_intensity = 1
+
+	/// The base duration increment, in deciseconds, for our dizziness to be increased by when we move.
+	var/head_movement_dizzy_base_duration = 1
+
+	/// The maximum time in deciseconds daze() may cause dizziness for
+	var/daze_dizziness_maximum_duration = 20 SECONDS
+
+	/// The maximum duration our nausea will last for. See _stomach.dm for the various levels of nausea.
+	var/max_nausea_duration = 20 SECONDS
+	/// The base amount of nausea we apply to our victim on movement.
+	var/chest_movement_base_nausea_score = 0.2 SECONDS
+	/// Percent chance, every time we move, to attempt to increase nausea of the victim if we are on the chest.
+	var/chest_movement_nausea_chance = 0
+
+	/// The minimum damage the chest must sustain before we try to increase their nausea.
+	var/chest_attacked_nausea_minimum_score = 7
+	/// Assuming we sustain more damage than our minimum, this is the chance for a given attack to proc a nausea attempt.
+	var/chest_attacked_nausea_chance = 25
+	/// Damage the chest takes is multiplied against this for determining the amount of nausea to apply.
+	var/chest_attacked_nausea_mult = 0.25 // saw = 15, 1.5 seconds of disgust at x1
+
+	/// Percent chance, every time we move, to attempt to damage random organs if we are on the chest.
+	var/chest_movement_organ_damage_chance = 0
+	/// The minimum total damage we can roll when doing random movement organ damage.
+	var/chest_movement_organ_damage_min = 1
+	/// The maximum total damage we can roll when doing random movement organ damage.
+	var/chest_movement_organ_damage_max = 3
+	/// The max amount of damage any specific organ can take from being randomly damaged on movement.
+	var/chest_movement_organ_damage_individual_max = 2
+
+	/// The max amount of damage any specific organ can take from being randomly damaged on attacked.
+	var/attacked_organ_damage_individual_max = 10
+	/// The chance for the internal organs of our limb to be damaged when the limb is attacked.
+	var/attacked_organ_damage_chance = 0
+	/// Score mult for overall organ damage on hit
+	var/attacked_organ_damage_mult = 0.5
+	/// Minimum score required to damage random organs on hit
+	var/attacked_organ_damage_minimum_score = 5
+
+	/// Our current counter for gel + gauze regeneration
+	var/regen_time_elapsed = 0 SECONDS
+	/// Time needed for gel to secure internals.
+	var/regen_time_needed = 30 SECONDS
+
+	/// If we have used bone gel to secure internals.
+	var/gelled = FALSE
```

Comment:
When I wrote this, I imagined bone gel as some sort of hardening glue that secured the bone. I guess we can switch it, but having it use bone gel also helps people get accustomed to it, since the ghetto treatment for bone wounds (blunt wounds) is already bone gel.